backend/src/main/java/com/bakdata/conquery/models/forms/util/DateContext.java

```diff
@@ -57,54 +58,48 @@
 	 */
 	@Getter @Setter
 	private LocalDate eventDate = null;
+	
+	/**
+	 * Indicates under which temporal subdivision mode this instance was created.
+	 */
+	@Getter
+	@Nullable
+	private DateContextMode subdivisionMode;
 
+	public static List<DateContext> generateAbsoluteContexts(CDateRange dateRangeMask, DateContextMode subdivisionMode) {
```

Comment:
wie häufig brauchst du den? Finde dass man das List.of auch in den Callern machen kann, wenn es nur 1-2 sind

Reply:
oder du machst ein ... draus